File: semestre-1/algoritmos/lista-1/python/MR-Alg-01-Ex-15.py
from math import tan, pi

l = float(input("Digite o comprimento do lado do polígono (em cm): "))
n = float(input("Digite o número de lados do polígono: "))

area = (n * l**2) / (4 * tan(pi / n))

print(f"Área do polígono regular de {n} lados = {area:.2f} cm²")

File: semestre-1/algoritmos/MR-Alg-01/.c/MR-Alg-01-Ex-15.c
#include <stdio.h>
#include <math.h>

#define PI 3.14159265358979323846

void main() {
    float l;
    int n;

    printf("Digite o comprimento do lado do polígono (em cm): ");
    scanf("%f", &l);

    printf("Digite o número de lados do polígono: ");
    scanf("%i", &n);

    float area = ((float)n * powf(l, 2)) / (4 * tanf(PI / (float)n));

    printf("Área do polígono regular de %i lados = %.2f cm²\n", n, area);
}